README.md

```diff
@@ -133,4 +142,4 @@ Contributions and testing by Jochen Christ, Martin Huber, Franziska Dessart, and
 Logo designed by [Sonja Scheungrab](https://twitter.com/multebaerr).
 
 <script async defer src="https://cdn.simpleanalytics.io/hello.js"></script>
-<noscript><img src="https://api.simpleanalytics.io/hello.gif" alt=""></noscript>
\ No newline at end of file
+<noscript><img src="https://api.simpleanalytics.io/hello.gif" alt=""></noscript>
```

Comment:
Did you remove the trailing line break?

Reply:
not intentionally. What is the problem with that?

Reply:
Git works best with a trailing line break at the end of each file.